dae-alright/src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,149 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
+
+
+@mixin footer-flex {
+    display: flex;
+    margin-left: auto;
+    margin-right: auto;
+    justify-content: space-between;
+    width: 90%; 
+}
+
+@mixin  footer-media-query($width){
+    @media screen and (max-width: $width){
+        @content;
+    }
+    
+}
+
+
+
+.deyAlrightFooterContainer{
+    padding: 0px;
+    margin: 0px;
+    box-sizing: border-box;
+    color: rgba(0, 0, 0, 0.4);
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+}
+select{
+    border: none;
+    color: rgba(0, 0, 0, 0.4);
+}
+
+.logoDiv{
+    //border: 3px solid black;
+    text-align: left;
+    padding-left: 150px;
+
+    @include footer-media-query(600px){
+        padding-left: 40px;
+    }
+
+    .footerLogo{
+        margin-left: 0;
+        //border: 2px solid chocolate;
+    }
+}
+
+
+
+.deyAlrightFooter{
+    padding-top: 25px;
+    padding-bottom: 80px;
+    width: 90%;
+    margin-left: auto;
+    margin-right: auto;
+    border-top: 1px solid #D6DB46;
+    border-bottom: 1px solid #D6DB46;
+
+
+    .deyAlrightFooterRow-1, .deyAlrightFooterRow-2{
+        @include footer-flex;
+        @include footer-media-query(600px){
+           flex-direction: column;
+        }
+
+
+        
+        .row1col1, .row2col1{
```

Comment:
Good job so far, I love what I'm seeing. But I think you can do better with your `class names`. When I see a CSS `classname`, I want to know what it's for just by looking at the name. Unfortunately, `row2col1`, `row1col2` all don't have that effect.

Reply:
class names have been adjusted to reflect what they do..... here is the commit link
https://github.com/devcareer/Dae-Alright-Frontend/pull/4/commits/67e309ab69263f98ed160d24871f1bfd48f3ad68